app/models/apple/config.rb

```diff
@@ -61,19 +60,20 @@ def self.find_or_build_private_feed(podcast)
       )
     end
 
+    # TODO: this a helper for onboarding via console, retrofit when the UX catches up
     def self.build_apple_config(podcast, key)
-      if has_apple_config?(podcast)
+      if podcast.apple_config
```

Comment:
Do we need this `has_apple_config?` method anymore?

Reply:
nope! will remove